src/server/adminBro.ts

```diff
@@ -1602,6 +1621,8 @@ const sendProjectsToGoogleSheet = async (
 
   // parse data and set headers
   const projectRows = projects.map((project: Project) => {
```

Comment:
I think we should set a filter for addresses that have `isRecipient:true` , because we may have more than two addresses ( a project can have multi related addresses but now just one or two of them are recipient)

Reply:
Oh ok, I thought since there was a restriction You couldn't have more than 2.
Ill add the filter for isRecipient

Reply:
Thanks Mohammad! I added the isRecipient filter in query and in the component.

Reply:
Thanks Carlos, I tested it and it works,